app/src/main/java/nodomain/freeyourgadget/gadgetbridge/export/GPXExporter.java

```diff
@@ -117,29 +119,53 @@ private boolean exportTrackPoint(XmlSerializer ser, ActivityPoint point, String
         if (description != null) {
             ser.startTag(NS_DEFAULT, "desc").text(description).endTag(NS_DEFAULT, "desc");
         }
-        ser.startTag(NS_DEFAULT, "src").text(source).endTag(NS_DEFAULT, "src");
+        //ser.startTag(NS_DEFAULT, "src").text(source).endTag(NS_DEFAULT, "src");
```

Comment:
Why did you remove the `src` element?

Reply:
Because a GPX file with 1000 track points would output "Amazfit BIP......." 1000 times.. It would make more sense to put the src in the header just once.  But just for expediency I simply commented it out.